overlord/fdestate/fdemgr.go

```diff
@@ -407,7 +412,7 @@ func (m *FDEManager) getRecoveryKey(keyID string) (rkey keys.RecoveryKey, err er
 // The state needs to be locked by the caller.
 func GetRecoveryKey(st *state.State, keyID string) (rkey keys.RecoveryKey, err error) {
```

Comment:
not really about the PR but is this actually going to be used? tasks will have access to the manager as well